osu.Server.Queues.ScoreStatisticsProcessor/Helpers/PlayValidityHelper.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Linq;
+using osu.Game.Online.API.Requests.Responses;
+using osu.Game.Rulesets;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Scoring;
+using osu.Server.Queues.ScoreStatisticsProcessor.Processors;
+
+namespace osu.Server.Queues.ScoreStatisticsProcessor.Helpers
+{
+    public static class PlayValidityHelper
+    {
+        /// <summary>
+        /// Whether the supplied <paramref name="score"/> is valid for tracking total play time and count.
+        /// </summary>
+        /// <seealso cref="PlayCountProcessor"/>
+        /// <seealso cref="PlayTimeProcessor"/>
+        /// <param name="score">The score to check.</param>
+        /// <param name="lengthInSeconds">The total length of play in seconds in the supplied <paramref name="score"/>.</param>
+        public static bool IsValidForPlayTracking(this SoloScoreInfo score, out int lengthInSeconds)
+        {
+            lengthInSeconds = GetPlayLength(score);
+
+            int totalObjectsJudged = score.Statistics.Where(kv => kv.Key.IsScorable()).Sum(kv => kv.Value);
+            int totalObjects = score.MaximumStatistics.Where(kv => kv.Key.IsScorable()).Sum(kv => kv.Value);
+
+            return lengthInSeconds >= 8
```

Comment:
I think this also needs a check for `Passed`, as that should always count.

Reply:
https://github.com/ppy/osu-queue-score-statistics/pull/188/commits/fad7b22dee148813f7c02d2cc3a59caa60d6c76d should do it I think?